src/openzaak/components/documenten/api/serializers.py

```diff
@@ -1163,3 +1163,12 @@ def create(self, validated_data):
         return ReservedDocument.objects.create(
             identificatie=identificatie, bronorganisatie=bronorganisatie
         )
+
+
+class ReserveerDocumentRequestSerializer(serializers.Serializer):
+    bronorganisatie = serializers.CharField(required=True)
+    amount = serializers.IntegerField(min_value=1, default=1, required=False)
+
+
+class ReservedDocumentListResponseSerializer(serializers.Serializer):
+    results = ReservedDocumentSerializer(many=True)
```

Comment:
the amount field can be added to original serializer, the ReservedDocumentListResponseSerializer is not needed, (see other comments)

the create method from the original serializer can also be removed since its done in the viewset